docs/hub/datasets-polars-operations.md

```diff
@@ -0,0 +1,105 @@
+# Transforming your dataset
+
+On this page we'll guide you through some of the most common operations used when doing data analysis. This is only a small subsection of what is possible in Polars, for more information please visit the [Documentation](https://docs.pola.rs/).
+
+For the example we will use the [Danbooru](https://huggingface.co/datasets/KBlueLeaf/danbooru2023-metadata-database) dataset. Danbooru is a large-scale anime image dataset with over 5 million images contributed and annotated in detail by an enthusiast community. Image tags cover aspects like characters, scenes, copyrights, artists, etc with an average of 30 tags per image.
```

Comment:
maybe you can showcase a more interesting dataset like the recent Common Crawl statistics at https://huggingface.co/datasets/commoncrawl/statistics ?